_includes/tutorials/kafka-consumer-application/kafka/markup/dev/make-dev-file.adoc

```diff
@@ -0,0 +1,19 @@
+Then create a development configuration file at `configuration/dev.properties`:
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-consumer-application/kafka/code/configuration/dev.properties %}</code></pre>
++++++
+
+Let's do a quick overview of some of the more important properties here:
+
+Deserilizers:: The `key.deserializer` and `value.deserializer` properties provide a class implementing the `Deserializer` interface for converting `byte` arrays into the expected object type of the key and value respectively.
+
+`max.poll.interval.ms`:: The `max.poll.interval.ms` is the maximum amount of time a consumer may take between calls to `Consumer.poll()`.  If a consumer instance takes longer than the specified time, it's considered non-reponsive and removed from the consumer-group triggering a rebalance.
+
+`enable.auto.commit`:: Setting this configuration to `true` enables the Kafka consumer to handle commiting offsets automatically for you.  The default setting is `true`, but it's included here to make it explicit.  When you enable auto commit, you need to ensure you've processed all records _**before**_ the consumer calls `poll` again.  Once there is a subsequent call to `poll`, all the records returned from the previous call are considered processed and the consumer commits the offsets.
+
+`auto.offset.reset`::  If a consumer instance can't locate any offsets for its topic-partition assignment(s), it will resume processing from the _**earliest**_ available offset.
+
+`group.id`:: Kafka uses the concept of a consumer-group which is used to represent a logical single group.  A consumer-group can be made up of mulitple members all sharing the same `group.id` configuration.  As members leave or join the consumer-group, the group-coordinator triggres a rebalance which causes topic-partion reassignment among active members of the group.
+
```

Comment:
The mix of the Header (Deserializers) and the formatted text as header looks a little odd in my opinion.  Maybe you could remove the headers from each and just reference the properties in each paragraph.  Possibly add bullet points if that would look good.

Reply:
Yep will do.  Good call.